CHANGELOG.md

```diff
@@ -40,6 +40,23 @@ All notable changes to this project will be documented in this file.
     * #### Removed
 * ### `nidcpower` (NI-DCPower)
     * #### Added
+        * API parity with NI-DCPower 2023 Q4.
+            * Properties added:
+                * `current_level_rising_slew_rate`
+                * `current_level_falling_slew_rate`
+                * `conduction_voltage_mode`
+                * `conduction_voltage_on_threshold`
+                * `conduction_voltage_off_threshold`
+                * `output_cutoff_voltage_measure_limit_high`
+                * `output_cutoff_voltage_measure_limit_low`
+            * Enums added:
+                * `ConductionVoltageMode`
+                * `EventOutputBehavior`
+                * `EventToggleInitialState`
+                * `LCRDCBiasTransientResponse`
+            * Enum values added:
+                * `NIDCPOWER_VAL_E_LOAD` added to enum `InstrumentMode`
```

Comment:
these are not the Python symbol names.

Reply:
Thanks for pointing this out, I've fixed these symbol names in the latest commit.